build-linux/pdagent.init

```diff
@@ -60,7 +60,7 @@ get_pid() {
 
 is_running() {
   pid=$(get_pid)
-  [ -n "$pid" ] && ps -p $pid | grep "$EXEC_NAME" >/dev/null 2>&1
+  [ -n "$pid" ] && ps $pid | grep "$EXEC_NAME" >/dev/null 2>&1
```

Comment:
Curious why -p is dropped?

Reply:
for some reason (we couldn't find documentation about why) `-p` wasn't actually displaying the full name of the process that was being run, so this no longer worked. for example:
`ps $pid` => `python /path/to/pdagent.py` => status: running
`ps -p $pid` => `python`=> status: not running

Reply:
got it, thank you!